ndb/src/google/cloud/ndb/model.py

```diff
@@ -16,7 +16,17 @@
 
 .. testsetup:: *
```

Comment:
Are these actively used by our test code or are they just here as documentation?

Reply:
They are actively used by the doctests. The doctests test the code snippets in the docs, but aren't considered part of the "real" test suite, though. They don't factor into coverage or anything like that.